graph-genome-previz.cwl

```diff
@@ -2,19 +2,24 @@
 cwlVersion: v1.0
 class: Workflow
 
+requirements:
+  ScatterFeatureRequirement: {}
+  StepInputExpressionRequirement: {}
+
 inputs:
   pangenome:
+    label: GFA1 or GF2 format
     type: File
-    # format: gfa
+    # format: GFA1 or GFA2
 
-  bin_width:
-    type: int
-    default: 1000
+  bin_widths:
+    type: int[]
+    default: [ 1, 4, 16, 64, 256, 1000, 4000, 16000]
     doc: width of each bin in basepairs along the graph vector
 
   cells_per_file:
     type: int
-    default: 5000 
+    default: 100
```

Comment:
Does it work in this default value for arvados data? (just to confirm)

Reply:
I don't understand the question, sorry. We have yet to automatically connect this workflow with the public sequence resource workflow. When that happens we can either keep this default value or set it.

Reply:
The workflow does complete using `100` for `cells_per_file` against the latest output from the public sequence results workflow, here are the results: https://workbench.lugli.arvadosapi.com/collections/lugli-4zz18-fml87xu0eaisn78

Reply:
Whoops, wrong output. See https://workbench.lugli.arvadosapi.com/collections/lugli-4zz18-e972zm26tlnxvmz